gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa/mfa-icons-resolver.ts

```diff
@@ -0,0 +1,37 @@
+export class MfaIconsResolver {
```

Comment:
it can be split to 2 enums and you can use the Enum types in getFactorIconType and getDisplayFactorType